runtime/doc/lua.txt

```diff
@@ -3670,4 +3670,87 @@ totable({f}, {...})                                       *vim.iter.totable()*
     Return: ~
         (table)
 
+
+==============================================================================
+Lua module: vim.snippet                                          *vim.snippet*
+
+active()                                                    *snippet.active()*
+    Returns `true` if there's an active snippet in the current buffer.
+
+    Return: ~
+        (boolean)
+
+cancel()                                                    *snippet.cancel()*
+    Cancels the current snippet.
+
+expand({input})                                             *snippet.expand()*
+    Expands the given snippet text.
+
```

Comment:
Maybe worth linking to the grammar definition, so users know what kind of input is accepted.